todo_app/data/trello_items.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import os
+from todo_app.data.item import Item
+
+def get_items():
+    url = "https://api.trello.com/1/boards/AbpSrXDQ/lists"
```

Comment:
Even though the board id (`AbpSrXDQ`) is not sensitive, it's still a good idea to read it from an environment variable, the same way you read the API key and token.

Because it is something you want to vary between environments. E.g. developers should test the app against different data from the "real" production board.